flake.nix

```diff
@@ -183,6 +183,8 @@
       morefine-m600 = import ./morefine/m600;
       msi-gs60 = import ./msi/gs60;
       msi-gl62 = import ./msi/gl62;
+      nxp-imx8mp-evk = import ./nxp/imx8mp-evk;
```

Comment:
What do I need to do here to get a bootable image for this hardware?

Reply:
Thanks. I have updated the document. Please have a look.
[ReadMe](https://github.com/gangaram-tii/nixos-hardware/blob/pr/imx8mp-evk-platform/nxp/README.md#22-for-imx8mq-evkimx8mp-evk)